We already include the patch for atk_bridge_set_event_context
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configure.ac
+++ b/configure.ac
@@ -27,7 +27,7 @@ m4_define([req_ver_atk], [1.18.0])
 
 PKG_CHECK_MODULES(JAW, [
   atk >= 2.14.0
-  atk-bridge-2.0 >= 2.33.1
+  atk-bridge-2.0
   atspi-2 >= 2.14.0
   dbus-1
   glib-2.0 >= 2.32.0
